jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/EtcdContainer.java

```diff
@@ -120,9 +133,13 @@ protected void configure() {
         withEnv("ETCD_LOG_LEVEL", this.debug ? "debug" : "info");
         withEnv("ETCD_LOGGER", "zap");
 
-        String user = System.getenv("TC_USER");
-        if (user != null) {
-            withCreateContainerCmdModifier(c -> c.withUser(user));
+        String tempUser = System.getenv("TC_USER");
+        if (this.user != null) {
+            tempUser = this.user;
+        }
```

Comment:
```suggestion
        String tempUser = this.user;
        if (tempUser != null) {
            tempUser = System.getenv("TC_USER");
        }
```

Reply:
@linghengqian I'll merge the pr once this suggestion is incorpporated

Reply:
- Done.